.changelog.yml

```diff
@@ -0,0 +1,20 @@
+---
+file_name: CHANGELOG.md
+excluded_labels:
+  - aakash
```

Comment:
was this intentional

Reply:
yes, still debugging to get all changelog.